src/app/components/InlineLink/index.tsx

```diff
@@ -0,0 +1,62 @@
+/** @jsx jsx */
+
+import { useContext, FC, HTMLAttributes } from 'react';
+import { jsx, Theme } from '@emotion/react';
+import Url from 'url-parse';
+
+import { FontVariant, GelFontSize } from '../../models/types/theming';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import Link from '../Link';
+import styles from './index.styles';
+
+interface Props extends HTMLAttributes<HTMLElement> {
+  allowCSR?: boolean;
+  className?: string;
+  fontVariant?: FontVariant;
+  size?: GelFontSize;
+  to: string;
+  text: string;
+}
+
+const bbcDomains = [
+  'localhost',
+  'www.bbc.com',
+  'bbc.com',
+  'www.bbc.co.uk',
+  'bbc.co.uk',
+  'www.bbcrussian.com',
+  'bbcrussian.com',
+];
+
+const InlineLink: FC<Props> = ({
+  allowCSR = false,
+  className,
+  fontVariant,
+  size,
+  text,
+  to,
+  ...htmlAttributes
+}: Props) => {
+  const { externalLinkText } = useContext(ServiceContext);
+  const { hostname } = new Url(to);
+  const isExternalLink = !bbcDomains.some(bbcDomain => hostname === bbcDomain);
+
+  return (
+    <Link
+      allowCSR={allowCSR}
+      className={className}
+      css={({ fontSizes, fontVariants }: Theme) => [
+        styles.self,
+        size && fontSizes[size],
+        fontVariant && fontVariants[fontVariant],
```

Comment:
Curious how the current inline-link handles setting its size and font variant, can't see anything obvious in `containers/InlineLink` that suggests what it does.

Reply:
The old `InlineLink` just [inherits styles](https://5d28eb3fe163f6002046d6fa-joijqnnhlk.chromatic.com/?path=/story/containers-inline-link--internal-link).

Happy to do the same with this one and remove the size and font variant so that it can be styled with the `css` prop only when necessary.

Reply:
I think it's ok to keep actually. Just thinking if this is a pattern we should adopt going forward, to allow all new text-based components to be able to set their font size/styling independently of the parent it may be a child of. Its optional I guess, so I'm happy for it to stay 👍🏼